app/utils/routing/utils.js

```diff
@@ -1,17 +1,24 @@
 import { URL_ROUTE_PREFIX_VBO, URL_ROUTE_PREFIX_LIG, URL_ROUTE_PREFIX_BRK } from './constants';
 
-export function getURL(object) {
+const getURL = object => {
+  const id = extractId(object);
+  if (!id) throw new Error(`Could not create url for given object ${JSON.stringify(object)}`);
+  return `/${id.type}/${id.id}/`;
+};
+
+const extractId = object => {
   const { vboId, ligId, brkId } = object;
```

Comment:
Might make sense to write a couple of small tests for this one, especially since it's a utility function that's used in multiple places.

Reply:
Done (36e4f473e2cb94c9065097fa0d3b638ab7b83175)